### DISCUSSION TITLE
npm search is down?

### DISCUSSION THREAD

**Discussion by shigma** [npm/feedback#742] [Website] [answered]
![image](https://user-images.githubusercontent.com/33423008/182234660-c71a9fa4-f920-4fb0-b236-de7034fbf77d.png)

but

![image](https://user-images.githubusercontent.com/33423008/182234695-033f8e0a-f5b6-4dee-bc86-02483dbf7ff8.png)

It seems that everything do not update after 2022-07-28.

**Comment by MylesBorins** [comment, marked as answer]
Thanks for sharing, we have an incident tracking this

https://status.npmjs.org/incidents/gqczj08nbrr9
